src/proj_config.h

```diff
@@ -0,0 +1,382 @@
+#ifndef PROJ_CONFIG_H
+#define PROJ_CONFIG_H
+
+#ifdef RTC_COCOA_FAMILY
+
+/* src/proj_config.h.  Generated from proj_config.h.in by configure.  */
+/* src/proj_config.h.in.  Generated from configure.ac by autoheader.  */
+
+/* define if the compiler supports basic C++11 syntax */
+#define HAVE_CXX11 1
```

Comment:
Skimming this file, I don't immediately see any differences in config from platform to platform. Do we even need these blocks after all?

Reply:
I believe it builds with an empty `proj_config.h`

Reply:
Apologies, I meant the RTC_ ifdefs. If they're all the same we can probably just remove the need to have them separated.